hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/MapTable.tpl

```diff
@@ -85,13 +85,12 @@
           </table>
         </div>
         <script type="text/javascript">
-          CRM.$(function($) {
-            {if $mappingDetailsError }
-              $('#saveDetails').show();
-            {else}
-              $('#saveDetails').hide();
-            {/if}
-          });
+          {if $mappingDetailsError }
+            cj('#saveDetails').show();
+          {else}
+            cj('#saveDetails').hide();
```

Comment:
I see.. so cj stands for CiviCRM Jquery. Well, in this case we could still use the $ variable, because it enclosed to the CRM.$ callback and would point to the jQuery instance loaded by CiviCRM.
Anyway, since you just copied it from the ContactMapTable.tpl, I think it is fine to keep it as this.